build-configuration/bom/pom.xml

```diff
@@ -77,6844 +77,1188 @@
-        <versionx.org.infinispan.infinispan-feature-pack-commons>${version.infinispan}</versionx.org.infinispan.infinispan-feature-pack-commons>
-        <versionx.org.infinispan.infinispan-feature-pack-embedded>${version.infinispan}</versionx.org.infinispan.infinispan-feature-pack-embedded>
-        <versionx.org.infinispan.infinispan-feature-pack-embedded-query>${version.infinispan}</versionx.org.infinispan.infinispan-feature-pack-embedded-query>
-        <versionx.org.infinispan.infinispan-feature-pack-server>${version.infinispan}</versionx.org.infinispan.infinispan-feature-pack-server>
-        <versionx.org.infinispan.infinispan-feature-pack-wf-modules>${version.infinispan}</versionx.org.infinispan.infinispan-feature-pack-wf-modules>
-        <versionx.org.infinispan.infinispan-hibernate-cache-commons>${version.infinispan}</versionx.org.infinispan.infinispan-hibernate-cache-commons>
-        <versionx.org.infinispan.infinispan-hibernate-cache-spi>${version.infinispan}</versionx.org.infinispan.infinispan-hibernate-cache-spi>
-        <versionx.org.infinispan.infinispan-hibernate-cache-v51>${version.infinispan}</versionx.org.infinispan.infinispan-hibernate-cache-v51>
-        <versionx.org.infinispan.infinispan-hibernate-cache-v53>${version.infinispan}</versionx.org.infinispan.infinispan-hibernate-cache-v53>
-        <versionx.org.infinispan.infinispan-javadoc>${version.infinispan}</versionx.org.infinispan.infinispan-javadoc>
-        <versionx.org.infinispan.infinispan-javadoc-all>${version.infinispan}</versionx.org.infinispan.infinispan-javadoc-all>
-        <versionx.org.infinispan.infinispan-javadoc-embedded>${version.infinispan}</versionx.org.infinispan.infinispan-javadoc-embedded>
-        <versionx.org.infinispan.infinispan-javadoc-remote>${version.infinispan}</versionx.org.infinispan.infinispan-javadoc-remote>
-        <versionx.org.infinispan.infinispan-jcache>${version.infinispan}</versionx.org.infinispan.infinispan-jcache>
-        <versionx.org.infinispan.infinispan-jcache-commons>${version.infinispan}</versionx.org.infinispan.infinispan-jcache-commons>
-        <versionx.org.infinispan.infinispan-jcache>${version.infinispan}</versionx.org.infinispan.infinispan-jcache>
-        <versionx.org.infinispan.infinispan-jcache-remote>${version.infinispan}</versionx.org.infinispan.infinispan-jcache-remote>
-        <versionx.org.infinispan.infinispan-js-client>${version.infinispan}</versionx.org.infinispan.infinispan-js-client>
-        <versionx.org.infinispan.infinispan-license>${version.infinispan}</versionx.org.infinispan.infinispan-license>
-        <versionx.org.infinispan.infinispan-lucene-directory>${version.infinispan}</versionx.org.infinispan.infinispan-lucene-directory>
+        <versionx.org.infinispan.elasticsearch-indexmanager-it>9.4.1-SNAPSHOT</versionx.org.infinispan.elasticsearch-indexmanager-it>
```

Comment:
Do we really need all these individual versions ? They bloat the pom and contribute nothing